packages/admin-panel/src/routes.js

```diff
@@ -121,6 +122,11 @@ export const ROUTES = [
         to: '/indicators',
         component: IndicatorsPage,
       },
+      {
+        label: 'Data-Tables',
```

Comment:
Do we like this naming style? With the `-`? I kinda like it as it makes clear that `Data-Tables` is a single concept, but it's not particularly convention with Tupaia to use `-`...

Reply:
I don't get it sorry 😆 
But it's just a label, - or no -, it's a + to me